JenkinsfileHW

```diff
@@ -42,15 +44,11 @@ lock(label: 'adgt_test_harness_boards') {
     harness.set_send_telemetry(false)
     harness.set_log_jira(false)
     harness.set_enable_resource_queuing(true)
-    harness.set_lock_agent(true) // Required for MATLAB toolbox tests
+    harness.set_lock_agent(false) // Required for MATLAB toolbox tests
```

Comment:
Out of curiosity, what is this change about? And do we need to update/remove the comment? Or do we indeed need the argument to be false for the MATLAB toolbox tests?

Reply:
The agent lock ensures that only one container is running at a time for MATLAB toolbox pipelines. This restriction was placed because of license errors when running MATLAB on multiple containers simultaneously. But now that the agents are using a network license, somehow the restriction is no longer necessary. I removed the comment but retained the setting.